it/src/main/java/com/google/cloud/teleport/it/bigtable/BigtableResourceManager.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (C) 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.it.bigtable;
+
+import com.google.cloud.bigtable.data.v2.models.Row;
+import com.google.cloud.bigtable.data.v2.models.RowMutation;
+import com.google.common.collect.ImmutableList;
+
+/** Interface for managing bigtable resources in integration tests. */
+public interface BigtableResourceManager {
+
+  /**
+   * Creates a Bigtable instance in which all clusters, nodes and tables will exist.
+   *
+   * @param clusters Collection of BigtableResourceManagerCluster objects to associate with the
+   *     given Bigtable instance.
+   * @throws BigtableResourceManagerException if there is an error creating the instance in
+   *     Bigtable.
+   */
+  void createInstance(Iterable<BigtableResourceManagerCluster> clusters)
+      throws BigtableResourceManagerException;
```

Comment:
nitpick: BigtableResourceManagerException is unchecked (RuntimeException), so no need to declare everywhere